### PR TITLE
Add-21_Merge_Two_Sorted_List

### PR DESCRIPTION
**Pull request by Rwatana**
Please check my code

### CODE REVIEW

File: 21_Merge_Two_Sorted_List.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# Definition for singly-linked list.
+# class ListNode:
+#     def __init__(self, val=0, next=None):
+#         self.val = val
+#         self.next = next
+# O(N) T(N)
+class Solution:
+    def mergeTwoLists(self, list1: Optional[ListNode], list2: Optional[ListNode]) -> Optional[ListNode]:
+        newHead = dummyHead = ListNode()
```
Comment (ryujumoto) [inline 21_Merge_Two_Sorted_List.py]: newHead -> new_head
dummyHead -> dummy_head

File: 21_Merge_Two_Sorted_List.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+from typing import ListNode
+from typing import Optional
+
+
+# Definition for singly-linked list.
+# class ListNode:
+#     def __init__(self, val=0, next=None):
+#         self.val = val
+#         self.next = next
+# O(N) T(N)
```
Comment (ryujumoto) [inline 21_Merge_Two_Sorted_List.py]: #time: O()
#space: O()

**Review by ryujumoto — COMMENTED** [review]
Done.

**Comment by Rwatana** [comment]
I revised them. Please check my code.

File: 21_Merge_Two_Sorted_List.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+
+# Definition for singly-linked list.
+# class ListNode:
+#     def __init__(self, val=0, next=None):
+#         self.val = val
+#         self.next = next
+class Solution:
+    def mergeTwoLists(self, list1: Optional[ListNode], list2: Optional[ListNode]) -> Optional[ListNode]:
+        new_Head = dummy_Head = ListNode()
```
Comment (ryujumoto) [inline 21_Merge_Two_Sorted_List.py]: new_Head -> new_head
dummy_Head -> dummy_head

**Comment by Rwatana** [comment]
I revised the code.
PLease check.

**Review by ryujumoto — APPROVED** [review]

**Pull request merged**